src/components/frame/HostConnector.tsx

```diff
@@ -30,7 +29,7 @@ export const HostConnector: FunctionComponent<HostConnectorProps> = ({ dispatch,
         dispatch(renderDocument({ document, rawDocument }));
       },
       selectTemplateTab: (tabIndex: number) => {
-        dispatch(selectTemplate(tabIndex));
+        dispatch(selectTemplate(tabIndex, {}));
```

Comment:
why is there a `{}` here?

Reply:
removed by keeping the templates internally